__tests__/api.test.js

```diff
@@ -87,6 +192,7 @@ describe("/api/articles/:article_id", () => {
             topic: expect.any(String),
             created_at: expect.any(String),
             votes: expect.any(Number),
+            comment_count: 11,
```

Comment:
Really we need to make a new test for this - to be following the principals of TDD. I know it feels a bit strange but please revert this test and add a new one specifically checking for the comment_count property.

Reply:
Done